cartographer_grpc/framework/server.h

```diff
@@ -75,8 +76,11 @@ class Server {
   };
   friend class Builder;
 
-  // Starts a server and waits for its termination.
-  void StartAndWait();
+  // Starts a server starts serving the registered services.
+  void Start();
+
+  // Initiates a server shutdown.
```

Comment:
"Initiates" sounds like this is asynchronous? When this method returns, the shut down is complete.

Reply:
Updated comment.